src/routes/contact-requests.ts

```diff
@@ -0,0 +1,32 @@
+import express, { Request, Response, Router } from 'express';
+import { ContactRequestsCollection } from '../collections/contact-requests';
+
+const contactRepository: ContactRequestsCollection = new ContactRequestsCollection();
+contactRepository.initContactRequestData();
+
+const router: Router = express.Router();
+export default router;
+
+// If a GET request is made to the "contact" route, return all saved contact requests.
+router.get('/', async (req: Request, res: Response) => {
+    const contactRequests = await contactRepository.returnAllContactRequests();
+    res.status(200).json(contactRequests);
+});
+
+// If a POST request is made to the "contact" route, save the supplied contact request.
+router.post('/', async (req: Request, res: Response) => {
+    const contactRequest = await contactRepository.addContactRequest(req.body);
+    res.status(201).json(contactRequest);
+});
+
+// If a DELETE request is made to the "clear" route, clear all data that is stored.
+// If the URL contains the query param ?reset=true, inject the default data to reset the collection!
+router.delete('/', async (req: Request, res: Response) => {
```

Comment:
Again, more 'colloquial' to clear with `delete: /` and, reset being an _additional_ action beyond basic CRUD/REST things - better to make it a flag on delete, since they share some commonality.

Reply:
Such a good call out! Thank you!